File: engine/engine-model/src/main/scala/tools/packet/transfer/name/NameChangeCancel.scala
package tools.packet.transfer.name

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class NameChangeCancel(private var _success: Boolean) extends PacketInput {
  def success: Boolean = _success

  override def opcode(): SendOpcode = SendOpcode.CANCEL_NAME_CHANGE_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/transfer/name/NameChangeCancel.java
package tools.packet.transfer.name;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record NameChangeCancel(Boolean success) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CANCEL_NAME_CHANGE_RESULT;
   }
}